pkg/api/v1/types.go

```diff
@@ -1475,6 +1475,27 @@ type PodLogOptions struct {
 	Previous bool `json:"previous,omitempty" description:"return previous terminated container logs; defaults to false"`
 }
 
+// PodAttachOptions is the query options to a Pod's remote attach call
+// TODO: merge w/ PodExecOptions below for stdin, stdout, etc
```

Comment:
I think duplication is fine in this case. Just put comments on both types that they should be kept in sync (i.e., add such a comment to PodExecOptions).